src/sentry/snuba/metrics/datasource.py

```diff
@@ -159,6 +159,12 @@ def get_metrics(projects: Sequence[Project], use_case_id: UseCaseID) -> Sequence
             "d": "generic_metrics_distributions",
             "g": "generic_metrics_gauges",
         },
+        "custom": {
+            "c": "generic_metrics_counters",
+            "s": "generic_metrics_sets",
+            "d": "generic_metrics_distributions",
+            "g": "generic_metrics_gauges",
```

Comment:
this absolutely doesn't need to slow down this change, but just a point of information: the metrics storage doesn't support gauges

Reply:
gotcha. i copied what was also declared above; would you prefer me to remove it for now?

we might eventually want to add gauges to generic metrics, but we can definitely get going without it.

Reply:
I think it would be fine if it remains but I will wait in case Oliver might have a different thought

Reply:
no, I wasn't asking you to remove it (I figured it was copy-pasted) -- just wanted to make sure that you were aware that gauges aren't there today